### ISSUE TITLE
Crash in XY Plot when processing Flux mode

### ISSUE THREAD

**Issue by wei-crypto** [TinyTerra/ComfyUI_tinyterraNodes#166]
escription:
The program crashes during AutoencodingEngine loading when using XY Plot to process multiple Flux LoRA files. The crash occurs during data type conversion from float8_e4m3fn to bfloat16.
Environment:
Python 3.10.11
PyTorch 2.4.1+cu124
CUDA 12.4
GPU: RTX 4060 Ti
ComfyUI version: aki-v1.4
Issue Details:
1. Loading and using Flux LoRA files individually works fine
Crashes when using XY Plot for batch processing
Crash occurs when loading the first LoRA file
Issue appears with both advanced xyPlot and advPlot images nodes
Error Log:
Starting server
To see the GUI go to: http://127.0.0.1:8188
G:\comfyui\ComfyUI-aki-v1.4\ComfyUI-aki-v1.4\custom_nodes\comfyui-mixlab-nodes\webApp\lib/photoswipe-lightbox.esm.min.js
FETCH DATA from: G:\comfyui\ComfyUI-aki-v1.4\ComfyUI-aki-v1.4\custom_nodes\ComfyUI-Manager\extension-node-map.json [DONE]
G:\comfyui\ComfyUI-aki-v1.4\ComfyUI-aki-v1.4\custom_nodes\comfyui-mixlab-nodes\webApp\lib/photoswipe.min.css
G:\comfyui\ComfyUI-aki-v1.4\ComfyUI-aki-v1.4\custom_nodes\comfyui-mixlab-nodes\webApp\lib/pickr.min.js
G:\comfyui\ComfyUI-aki-v1.4\ComfyUI-aki-v1.4\custom_nodes\comfyui-mixlab-nodes\webApp\lib/classic.min.css
G:\comfyui\ComfyUI-aki-v1.4\ComfyUI-aki-v1.4\custom_nodes\comfyui-mixlab-nodes\webApp\lib/model-viewer.min.js
G:\comfyui\ComfyUI-aki-v1.4\ComfyUI-aki-v1.4\custom_nodes\comfyui-mixlab-nodes\webApp\lib/juxtapose.min.js
G:\comfyui\ComfyUI-aki-v1.4\ComfyUI-aki-v1.4\custom_nodes\comfyui-mixlab-nodes\webApp\lib/juxtapose.css
QualityOfLifeSuit_Omar92::NSP ready
['flux1-canny-dev-lora.safetensors', 'flux1-depth-dev-lora.safetensors']
['flux1-canny-dev-lora.safetensors', 'flux1-depth-dev-lora.safetensors']
got prompt
Using xformers attention in VAE
Using xformers attention in VAE
Requested to load FluxClipModel_
Loading 1 new model
loaded completely 0.0 4777.53759765625 True
clip missing: ['text_projection.weight']
G:\comfyui\ComfyUI-aki-v1.4\ComfyUI-aki-v1.4\comfy\ldm\modules\attention.py:407: UserWarning: 1Torch was not compiled with flash attention. (Triggered internally at C:\actions-runner\work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\transformers\cuda\sdp_utils.cpp:555.)
out = torch.nn.functional.scaled_dot_product_attention(q, k, v, attn_mask=mask, dropout_p=0.0, is_causal=False)
(URLError(gaierror(11004, 'getaddrinfo failed')),)
model weight dtype torch.float8_e4m3fn, manual cast: torch.bfloat16
model_type FLUX
Requested to load Flux
Loading 1 new model
loaded completely 0.0 11350.048889160156 True
Requested to load AutoencodingEngine
Stack Trace:
Windows fatal exception: access violation
Stack (most recent call first):
File "G:\comfyui\ComfyUI-aki-v1.4\ComfyUI-aki-v1.4\python\lib\site-packages\torch\storage.py", line 354 in getitem
File "G:\comfyui\ComfyUI-aki-v1.4\ComfyUI-aki-v1.4\python\lib\site-packages\safetensors\torch.py", line 315 in load_file
File "G:\comfyui\ComfyUI-aki-v1.4\ComfyUI-aki-v1.4\comfy\utils.py", line 34 in load_torch_file
File "G:\comfyui\ComfyUI-aki-v1.4\ComfyUI-aki-v1.4\comfy\sd.py", line 694 in load_diffusion_model
File "G:\comfyui\ComfyUI-aki-v1.4\ComfyUI-aki-v1.4\nodes.py", line 882 in load_unet
File "G:\comfyui\ComfyUI-aki-v1.4\ComfyUI-aki-v1.4\custom_nodes\ComfyUI_tinyterraNodes\ttNpy\ttNexecutor.py", line 209 in map_node_over_list
Attempted Solutions:
Tried different versions of ComfyUI_tinyterraNodes (including the latest)
2. Tested different Plot nodes
Verified model file integrity
Expected Behavior:
XY Plot should be able to process multiple Flux LoRA files without crashing.

**Comment by TinyTerra** [comment]
Do you have a sample workflow i can use to test this against?
